agent/php_user_instrument_op_array.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if LOOKUP_METHOD == LOOKUP_USE_OP_ARRAY
```

Comment:
This module/code is verbatim copy of https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_user_instrument.c#L647-L732. It's been factored out into lookup method implementation specific module for easier testing and is enabled at compile time for PHP < 7.4 as configured in `agent/php_user_instrument_lookup_config.h`